crt_portal/cts_forms/views.py

```diff
@@ -715,13 +715,17 @@ def get_batch_data(disposition_batches, all_args_encoded):
 
 
 def get_batch_view_data(request):
+    # clear out any batches that were not completed
+    empty_batches = ReportDispositionBatch.objects.filter(disposed_count=0)
+    for batch in empty_batches:
+        batch.delete()
```

Comment:
```suggestion
    # clear out any batches that were not completed
    ReportDispositionBatch.objects.filter(disposed_count=0).delete()
```

it should be possible to call delete on a queryset, which will result in one query for everything instead of one for each batch

https://docs.djangoproject.com/en/5.1/ref/models/querysets/#delete